src/grub.cfg

```diff
@@ -40,10 +40,21 @@ else
   set timeout=1
 fi
 
+# Determine if this is a first boot and set the ${ignition_firstboot} variable
```

Comment:
So the ignition-dracut fragment is dead code now?  And really in general, everything in `/etc/grub.d`?

Reply:
Essentially, yes. I opened https://github.com/coreos/ignition-dracut/pull/91 to make that clear. But yeah, there's a bunch of stuff left in there.

I guess we could nuke it all in postprocessing? Though it should eventually drop out anyway if we continue on the no-grub2-utilities-on-disk path.

Reply:
Can't nuke it yet because we need it for the architectures that still use Anaconda right?